src/vss_tools/vspec/vssexporters/vss2samm/config.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2024 Contributors to COVESA
+#
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License 2.0 which is available at
+# https://www.mozilla.org/en-US/MPL/2.0/
+#
+# SPDX-License-Identifier: MPL-2.0
+
+# General CONFIG variables
+# Custom string, which we use to escape " and ' characters in VSS node description/comments
+# Used in file_helper.write_graph_to_file to properly escape characters in filedata, before to write it to a file.
+
+
+class Config:
```

Comment:
The Config file must be initialized in the vss2samm.py **and** before any other helper, which is using it, to be loaded.

Otherwise, when helpers are loaded **before** the initialization of the config, the OUTPUT_NAMESPACE, VSPEC_VERSION and SPLIT_DEPTH will be None, which will cause problems like: `@prefix : <urn:samm:None:None#> .`  in the generated aspect models.

Expected result should be like: `@prefix : <urn:samm:com.covesa.vss.spec:5..0.0#> .`

This can be observed in the below comments.